docs/API.md

```diff
@@ -345,6 +345,27 @@ Called when a route is about to be entered. It provides the next router state an
 
 If `callback` is listed as a 3rd argument, this hook will run asynchronously, and the transition will block until `callback` is called.
 
+###### `callback` signature
+`cb(err)`
+
+```js
+const userIsInAnOrganisation = (nextState, replace, callback) => {
```

Comment:
Sorry for the awful tiny nit, but I think we generally use AmE for docs, to the extent it matters. Maybe this could just be `userIsInAnOrg`?